samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1967,7 +1969,7 @@ static void run(
         queryLab.createJoinTables(dbAdminClient, database);
     break;
       case "queryLabInsertTestDataToJoinTables":
-        queryLab.insertTestDataToJoinTables(dbClient, 50, 100000);
+        queryLab.insertTestDataToJoinTables(dbClient, MUTATIONS_PER_TRANSACTION, /*Rows to be inserted*/ 100000);
```

Comment:
a) this should be consistent. We should either have everything as constants, or have everything as inline comments.
b) the format for an inline comment is the name of the variable. If the comment is longer than that, it should not be in-lined. Here I think it would be /*numRows*/